2024-09-29 22:41:54,193 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Configure stats pid to 37104
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Loading settings from /home/egor/.config/wandb/settings
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Loading settings from /home/egor/desktop/tmp/precondnn/exps/loss_choice_exps/exp02/wandb/settings
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_init.py:_log_setup():524] Logging user logs to /home/egor/desktop/tmp/precondnn/exps/loss_choice_exps/exp02/wandb/run-20240929_224154-yvpn4w4o/logs/debug.log
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_init.py:_log_setup():525] Logging internal logs to /home/egor/desktop/tmp/precondnn/exps/loss_choice_exps/exp02/wandb/run-20240929_224154-yvpn4w4o/logs/debug-internal.log
2024-09-29 22:41:54,194 INFO    MainThread:37104 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff8f7676f10>
2024-09-29 22:41:54,195 INFO    MainThread:37104 [wandb_init.py:init():607] calling init triggers
2024-09-29 22:41:54,195 INFO    MainThread:37104 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-29 22:41:54,195 INFO    MainThread:37104 [wandb_init.py:init():657] starting backend
2024-09-29 22:41:54,195 INFO    MainThread:37104 [wandb_init.py:init():661] setting up manager
2024-09-29 22:41:54,196 INFO    MainThread:37104 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-09-29 22:41:54,197 INFO    MainThread:37104 [wandb_init.py:init():669] backend started and connected
2024-09-29 22:41:54,209 INFO    MainThread:37104 [wandb_run.py:_label_probe_notebook():1344] probe notebook
2024-09-29 22:41:54,209 INFO    MainThread:37104 [wandb_init.py:init():767] updated telemetry
2024-09-29 22:41:54,230 INFO    MainThread:37104 [wandb_init.py:init():800] communicating run to backend with 90.0 second timeout
2024-09-29 22:41:54,847 INFO    MainThread:37104 [wandb_init.py:init():851] starting run threads in backend
2024-09-29 22:42:00,547 INFO    MainThread:37104 [wandb_run.py:_console_start():2463] atexit reg
2024-09-29 22:42:00,547 INFO    MainThread:37104 [wandb_run.py:_redirect():2309] redirect: wrap_raw
2024-09-29 22:42:00,548 INFO    MainThread:37104 [wandb_run.py:_redirect():2374] Wrapping output streams.
2024-09-29 22:42:00,548 INFO    MainThread:37104 [wandb_run.py:_redirect():2399] Redirects installed.
2024-09-29 22:42:00,550 INFO    MainThread:37104 [wandb_init.py:init():894] run started, returning control to user process
2024-09-29 23:26:50,867 INFO    MainThread:37104 [wandb_run.py:_finish():2160] finishing run cmcmsu/loss_choice_02/yvpn4w4o
2024-09-29 23:26:50,867 INFO    MainThread:37104 [jupyter.py:save_history():444] not saving jupyter history
2024-09-29 23:26:50,868 INFO    MainThread:37104 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-09-29 23:26:50,868 INFO    MainThread:37104 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-09-29 23:26:50,868 INFO    MainThread:37104 [wandb_run.py:_atexit_cleanup():2424] got exitcode: 0
2024-09-29 23:26:50,868 INFO    MainThread:37104 [wandb_run.py:_restore():2406] restore
2024-09-29 23:26:50,869 INFO    MainThread:37104 [wandb_run.py:_restore():2412] restore done
2024-09-29 23:26:50,869 INFO    MainThread:37104 [wandb_run.py:_on_finish():2677] communicating current version
2024-09-29 23:26:51,189 INFO    MainThread:37104 [wandb_run.py:_on_finish():2686] got version response upgrade_message: "wandb version 0.18.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-09-29 23:26:57,398 INFO    MainThread:37104 [wandb_run.py:_footer_history_summary_info():4078] rendering history
2024-09-29 23:26:57,398 INFO    MainThread:37104 [wandb_run.py:_footer_history_summary_info():4110] rendering summary
2024-09-29 23:26:57,411 INFO    MainThread:37104 [wandb_run.py:_footer_sync_info():4037] logging synced files
